src/main/java/info/sjd/controller/OrderController.java

```diff
@@ -59,12 +59,11 @@
     }
 
     @GetMapping(path = "update_amount")
-    public ResponseEntity<Order> updateAmount(@RequestParam Integer id, Integer amount){
-        Order order = orderService.updateAmount(id, amount);
-        if (order != null){
-            return new ResponseEntity<>(orderService.updateAmount(id,amount), HttpStatus.OK);
+    public ResponseEntity<Order> updateAmount(@RequestParam Integer id,@RequestParam Integer amount){
```

Comment:
The same situation as before in updateStatus method

Reply:
done